src/main/java/org/fundacion/jala/converter/controller/ProjectController.java

```diff
@@ -20,29 +21,32 @@
 import org.springframework.web.bind.annotation.PostMapping;
 import java.io.IOException;
 
+/**
+ * This class creates a project and saves it in the database.
+ */
 @RestController
 @RequestMapping("/api")
 public class ProjectController {
 
     /**
-     * Endpoint for creating a project in data base
+     * Endpoint for creating a project in data base.
+     *
```

Comment:
Is missing JavaDoc about Params (@params) and exception (@throws)

Reply:
added.